flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManager.java

```diff
@@ -141,7 +133,7 @@ private Map<String, HadoopDelegationTokenProvider> loadProviders() {
     @VisibleForTesting
     boolean isProviderEnabled(String serviceName) {
         return configuration.getBoolean(
-                String.format("security.kerberos.token.provider.%s.enabled", serviceName), true);
+                String.format("security.delegation.token.provider.%s.enabled", serviceName), true);
```

Comment:
Do we have a documentation for these configurations somewhere? Do we need to update it?

Reply:
The PR is there but not yet merged. https://github.com/apache/flink/pull/21192 Everybody is watching @mbalassi 's hand :)